### PR TITLE
Create a slide to list Go Concurrency topics and Concurrency in Go Slide

### PR DESCRIPTION
**Pull request by linuxmonk**
This PR has 3 slides

- Header
- Topics
- Concurrency in Go

**Topics**
Concurrency
Concurrency in Go
Goroutines
Channels
Range & Select
Under the hood: Processes, Threads, Goroutines
Under the hood: Go Runtime

https://pr82-dot-gotraining-testing.appspot.com/gocourse-2019-05-melbourne.slide#73

### CODE REVIEW

File: content/gocourse-2019-05-melbourne.slide [hunk 1]
```diff
@@ -841,31 +841,18 @@ Embedding types introduces the possiblity for naming conflicts. Go follows some
 - Bonus points: Add a third `Storer` implementation using [[https://github.com/syndtr/goleveldb][LevelDB]]
 
 
-* Concurrency (1/2)
+* Go Concurrency
 
-"Don't communicate by sharing memory, share memory by communicating."
```
Comment (anzboi) [inline content/gocourse-2019-05-melbourne.slide]: I feel like these first four dot-points still have a place here
They talk about the mindset the creators had going into developing go's concurrency model.

IMO add a slide after the concurrency title slide and before the topics.
slide title suggestion: 'Concurrency in go'

**Review by anzboi — COMMENTED** [review]
I feel like we're leaving out some interesting insights and useful links, see comment for more detail.

File: content/gocourse-2019-05-melbourne.slide (hunk 1 above)
Comment (juliaogris) [reply content/gocourse-2019-05-melbourne.slide]: Yup, I agree with that.

Comment (linuxmonk) [reply content/gocourse-2019-05-melbourne.slide]: I'll do that

Comment (linuxmonk) [reply content/gocourse-2019-05-melbourne.slide]: Done

**Comment by linuxmonk** [comment]
Based on today's discussion the revised topic list -

- Go routines `go`
- Channels
- Range & Select

Further synchronisation tools: Mutex, Wait Group
Under the hood: Processes, threads, goroutines, go Scheduler

File: content/gocourse-2019-05-melbourne.slide [hunk 2]
```diff
@@ -910,37 +910,40 @@ Embedding types introduces the possiblity for naming conflicts. Go follows some
...
-- [[https://goplay.space/#R1qgjOsPSGk][Select]] ([[https://play.golang.org/p/R1qgjOsPSGk][Playground]]`*`)
-- [[https://goplay.space/#4TGM29YbvX4][Deadlocks]] ([[https://play.golang.org/p/4TGM29YbvX4][Playground]]`*`)
+- "Don't communicate by sharing memory, share memory by communicating."
+- "Concurrency is not parallelism."
+- "Concurrency is about structure, parallelism is about execution."
 
-`*` The [[https://play.golang.org/][official Go Playground]] prints in real-time with delays but lacks syntax colouring.
+* Concurrency in Go
 
+- Go provides:
```
Comment (camscale) [inline content/gocourse-2019-05-melbourne.slide]: nit: this should not be a bullet point, just a bare line

**Review by camscale — DISMISSED** [review]
LGTM. just one minor nit.

**Review by camscale — APPROVED** [review]
LGTM

**Pull request merged**